index.html

```diff
@@ -474,9 +480,22 @@ <h2 class="bannerbox__title">Can I help some other way?</h2>
                                     <input type="checkbox" name="Capability" value="tech"/> Contribute my tech wizardry
                                 </label>
                                 <label>
+                                    <input type="checkbox" name="Capability" value="recruit"/> Help recruit and organize volunteers
+                                </label>
+                                <label>
                                     <input type="checkbox" name="Capability" value="raise money"/> Help the project raise money
                                 </label>
+                                <label>
+                                    <input type="checkbox" name="Capability" value="other"/> Other (please specify)
+                                    <br/>
+                                    <input type="text" class="form-input" id="helpOther" placeholder="Other" name="other">
+                                    <div class="help-block with-errors"></div>
+                                </label>
                             </div>
+                            <br/>
+                            <p>We also need funding to keep building the movement. Our volunteers are working around the clock but some things cannot be done with time and love alone. Could you spare some money to contribute to the campaign?</p>
+                            <p><a href="https://www.generosity.com/fundraising/CarpoolVote-com-linking-volunteer-drivers-voters" target="_blank">Support our IndieGogo campaign</a></p>
+                            <p>Or get in touch at: <a href="mailto:donations@CarpoolVote.com">donations@CarpoolVote.com</a></p>
```

Comment:
`@CarpoolVote.com` should probably be lowercase, but I'm not sure that it matters.